extensions/wikibase/module/scripts/schema-alignment.js

```diff
@@ -1557,7 +1580,7 @@ SchemaAlignment._initField = function(inputContainer, mode, initialValue, change
                 initialValue.type === "wbdatevariable" ||
                 initialValue.type === "wblocationvariable" ||
                 initialValue.type === "wblanguagevariable") {
-        var cell = SchemaAlignment._createDraggableColumn(initialValue.columnName, false);
+        var cell = SchemaAlignment._createDraggableColumn(initialValue.columnName, false, null);
```

Comment:
Instead of passing `null` here we could potentially try to look up the column by name and get its recon config. That would let us show logos on columns which are inserted in the schema.

Reply:
Oh yes, fixing this asap!